examples/QmlBarcodeReader/qml/GeneratorPage.qml

```diff
@@ -0,0 +1,254 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtMultimedia 5.12
+import QtQuick.Layouts 1.12
+import com.scythestudio.sbarcodereader 1.0
+
+Item {
+    id: root
+    anchors.fill: parent
+
+    SBarcodeGenerator {
+        id: barcodeGenerator
+        onProcessFinished: {
+            console.log(barcodeGenerator.filePath);
+            image.source = "file:///" + barcodeGenerator.filePath
+        }
+    }
+
+    Rectangle {
+        anchors.fill: parent
+        height: parent.height
+        width: parent.width
+        visible: true
+        color: "white"
```

Comment:
White colors is default, visible = true is default, you don't need to set height and width once you set anchors.fill: parent.